src/directives/for-of/for_of.directive.ts

```diff
@@ -48,10 +48,10 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
     @Input() public igxForRemote = false;
 
     @Output()
-    public onChunkLoaded = new EventEmitter<any>();
+    public onItemLoad = new EventEmitter<any>();
 
     @Output()
-    public onChunkLoading = new EventEmitter<any>();
+    public onItemPreload = new EventEmitter<any>();
```

Comment:
@skrustev Is it only one item that gets preloaded? If they are multiple then the API should be `onItemsPreload`.